DoctrineBundle.php

```diff
@@ -133,24 +133,4 @@ public function shutdown()
             }
         }
     }
-
-    /**
-     * {@inheritDoc}
-     */
-    public function registerCommands(Application $application)
```

Comment:
I think this needs to still be implemented, in order to prevent calling the parent.
Otherwise, when the ORM is not loaded, this will still register the related commands, using the legacy way.

Reply:
hmm, indeed

Reply:
Indeed. Done.